README.md

```diff
@@ -14,6 +15,191 @@ The NVIDIA MONAI Cloud APIs offer a comprehensive suite of tools designed to enh
+sudo rm -f /usr/share/orthanc/plugins/*.so
+
+sudo wget https://lsb.orthanc-server.com/orthanc/1.9.7/libServeFolders.so --output-document /usr/share/orthanc/plugins/libServeFolders.so
+sudo wget https://lsb.orthanc-server.com/orthanc/1.9.7/libModalityWorklists.so --output-document /usr/share/orthanc/plugins/libModalityWorklists.so
+sudo wget https://lsb.orthanc-server.com/plugin-dicom-web/1.6/libOrthancDicomWeb.so --output-document /usr/share/orthanc/plugins/libOrthancDicomWeb.so
+
+sudo service orthanc restart
+```
+
+```
+dicom_web_endpoint = ...
+dicom_client_id = ...
+dicom_client_secret = ...
+```
+
+After setup, you can use the Orthanc browser located at http://127.0.0.1:8042/app/explorer.html#upload to upload files.
+if the dicomweb needs a credentials setup, please refer to [Orthanc Guide](https://orthanc.uclouvain.be/book/users/configuration.html)
+Open the configuration.json and change credentials: 
+```
+// Whether or not the password protection is enabled
+"AuthenticationEnabled" : true,
+
+// The list of the registered users. Because Orthanc uses HTTP
+// Basic Authentication, the passwords are stored as plain text.
+"RegisteredUsers" : {
+  "user": "pass"
+}
+```
+
+### Basic OHIF Instruction
+
+
+#### 1. **Study List**
+Open OHIF viewer, the index page is a study list. The subjects listed are prepared in dicom-web storage. 
+
+![study list](docs/doc_imgs/studyList.png)
+
+#### 2. **Select Subject** 
+
+Click any subject you would like to annotate. 
+The dropdwon panel will show several options. Select the `MONAI Service` 
+to load the NVIDIA MONAI Cloud API plugin.
+
+![Select an image](docs/doc_imgs/selectanimage.png)
+
+If you're using the API directly, you can use the `nextimage` endpoint.
+
+#### 3. **Selected MONAI Service Plugin**
+
+Click the MONAI Service plugin button on the right panel after the tri-planer views are up. 
+The MONAI Service planel will show on the right, the plugin will automatically connect
+to the running MONAI Service server and to load executed models and created datasets. 
+![allclass](docs/doc_imgs/viewerpage.jpg)
+
+
+
+#### 4. **Run Inferencing Using Selected Method**
+
+Below is a VISTA model application example demo:
+VISTA model support multiple AI inference strategies.  
+1. **Segment All Classes**: Predict all trained anatomy targets in the VISTA model. 
+2. **Using Class Prompts**: Select and predict one or multiple anatomies with VISTA model.
+3. **Using Point Prompts**: Place control point and edit the anatomy with click prompts. 
+
+Once you've picked your preferred method, run the inference to get an initial annotation.
+
+![allclass](docs/doc_imgs/annotation.png)
+
+#### 5. **Annotate / Refine Annotations use side panel **
+
+With the initial mask in place, you might notice areas that require manual tweaking. 
+Use the provided annotation tools to:
+
+- Refine boundaries
+- Add or remove regions
+
+This step ensures that your annotations are as accurate as possible.
```

Comment:
This step is crucial to ensure the highest possible accuracy of your annotations.